modules/claims_api/app/services/claims_api/slack/failed_submissions_messenger.rb

```diff
@@ -93,9 +93,11 @@ def build_error_block(title, errors)
         }
       end
 
-      def link_value(id)
+      def link_value(id, type = :eid)
         return 'N/A' if id.blank?
 
+        return 'N/A' if type == :tid && TID_SUBSTRING_WHITELIST.none? { |substr| id.upcase.include? substr }
+
```

Comment:
@cilestin When I test this I get a 500
```
"exception": "uninitialized constant ClaimsApi::Slack::FailedSubmissionsMessenger::TID_SUBSTRING_WHITELIST",
```
<img width="1969" alt="Screenshot 2025-03-19 at 8 17 32 AM" src="https://github.com/user-attachments/assets/e8fe8d2c-3246-43fe-afe9-61e35b1ed0ff" />

Reply:
Commit [dd26d4e](https://github.com/department-of-veterans-affairs/vets-api/pull/21235/commits/dd26d4ebe3227c8951c9d7e0d17c84ff3142aba6) fixes the missing constant, [63bb8e5](https://github.com/department-of-veterans-affairs/vets-api/pull/21235/commits/63bb8e57fa16f30d72c3f6babff8bb1c75ee686c) extracts the tag portion of the TID.